test/integration/reasoner/planner/RecursivePlannerTest.java

```diff
@@ -104,7 +106,7 @@ public void tearDown() {
     @Test
     public void test_single_retrieval() {
         initialise(Arguments.Session.Type.DATA, Arguments.Transaction.Type.READ);
-        RecursivePlanner planSpaceSearch = RecursivePlanner.create(transaction.traversal(), transaction.concepts(), transaction.logic());
+        RecursivePlanner planSpaceSearch = RecursivePlanner.create(transaction.traversal(), transaction.concepts(), transaction.logic(), new ReasonerPerfCounters(false));
```

Comment:
This is a nicer signature than passing a random boolean in with a bunch of higher-level components!

Reply:
That was on the other branch. It will still be re-introduced after the rebase
.